scalableminds/webknossos
# Enable multiline comments

This PR properly enables the multiline comments, adds escaping for newlines to the frontend nml serializer and adds tests for those multiline nmls.
@fm3 Feel free to have a look and/or test :)

### URL of deployed dev instance (used for testing):
- https://enablemultilinecomments.webknossos.xyz

### Steps to test:
- Create a tracing. Create multiline comments using the `Edit Markdown` button in the comment tab, or by entering `\n` in the comment edit field.
- Download/Upload the tree with multiline comments using all combinations of frontend/backend download/upload. Newlines should be preserved properly.

### Issues:
- fixes #2824 
- fixes #2991 

------
